pkg/xds/generator/testdata/ingress/04.envoy.golden.yaml

```diff
@@ -135,6 +135,7 @@ resources:
         - filterChainMatch:
             serverNames:
               - backend{mesh=mesh1,version=v2}
+            transportProtocol: tls
```

Comment:
did you fix this manually? Please use "Golden machinery" next time. It can fix the test and you will save time for the next person